numismatic/feeds/bravenewcoin.py

```diff
@@ -0,0 +1,47 @@
+from itertools import product
+import attr
+from .base import Feed
+from ..libs.config import get_config
+
+class BraveNewCoin(Feed):
+    config = get_config()['bravenewcoin'] if 'bravenewcoin' in get_config() \
+        else {}
+    api_key_id = (config.get('api_key_id', '') if config else '')
+    api_key_secret = (config.get('api_key_secret', '') if config else '')
```

Comment:
The api credentials should be instance variables.

Reply:
I am happy with this. A bigger question is if you are happy with putting the config logic here? I think it is better here than in the cli.py

If you are happy, then I will have the final PR (with all your comments) available to you tonight....